src/fs/rename.js

```diff
@@ -1,5 +1,18 @@
+import fs from 'fs';
+import path from 'path';
+import {fileURLToPath} from 'url';
+
+const __filename = fileURLToPath(import.meta.url);
+
+const __dirname = path.dirname(__filename);
+
+
 const rename = async () => {
     // Write your code here 
+    
+    fs.rename(__dirname + '/files/wrongFilename.txt', __dirname + '/files/properFilename.md', (error) => {
```

Comment:
There is no check whether `properFilename.md` file exist in the folder. Deso not meet requirement (properFilename.md already exists Error with message FS operation failed must be thrown)